MetaMask/metamask-extension
# feat: add NumBlock network and token icons

feat: add NumBlock network and token icons




## **Description**

Add NumBlock mainnet network icon and token icon support.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25249?quickstart=1)

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Go to [NumBlock Mainnet Explorer](https://mainnet.numblock.org/)
2. Scroll down to bottom and click the Metamask icon to add network
3. Open Metamask extension to check the network and token icons

## **Screenshots/Recordings**


### **Before**

<img width="361" alt="Screenshot 2024-06-12 at 15 43 10" src="https://github.com/MetaMask/metamask-extension/assets/5998334/eb985f63-4502-4038-a527-605d52b3a254">

### **After**

![image](https://github.com/MetaMask/metamask-extension/assets/5998334/d35e0486-8053-4cf3-9e87-b7c0e1ef0b81)



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
